workflows/update_tags.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+# MIT License
+#
+# (C) Copyright 2021-2023 Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+
+# This script updates all image tag references in the workflows directory.
+# This is necessary since a new docs-csm version could land before or after a CSM update.
+
+set -e
+REGISTRY="registry.local"
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+cd $SCRIPT_DIR
+
+if [[ "$(basename ${SCRIPT_DIR})" != "workflows" ]]; then
+  echo "ERROR: This script must be located in the workflows directory to run."
+  exit 1
+fi
+
+if [[ $(nslookup $REGISTRY >> /dev/null; echo $?) != 0 ]]; then
+  echo "INFO: Skipping update_tags.sh script given docs were installed outside of a CSM environment."
+  exit 0
+fi
+
+if [[ ! $(curl -s https://${REGISTRY}) ]]; then
+  echo "ERROR: Is this a CSM environment? Check that ${REGISTRY} is accessible over https before continuing."
```

Comment:
I think this will cause a misleading failure when docs-csm is installed on the PIT node.

Reply:
In conversation, before this, check for /etc/pit-release existence and exit with message that this doesn't run on PIT.

Reply:
fixed